### DISCUSSION TITLE
Implementing Clickhuse DB for Coroot

### DISCUSSION THREAD

**Discussion by T100D** [coroot/coroot#172] [Q&A] [answered]
Hi All,

For a proof of concept we am doing a testdrive with Coroot (latest version) that is very promising so far.

How to create a clickhouse DB locally that is accessible for Coroot.

Steps so far:
docker run -d -p 18123:8123 -p9001:9000 --name some-clickhouse-server --ulimit nofile=262144:262144 clickhouse/clickhouse-server

in the container added a password for tje default user testing proves it is accessible and Coroot does not work without.

Configuring clickhouse in Coroot on native access shows us a error "code: 60, message: Table default.otel_traces does not exist"

So how to create the tables in Clisckhouse to get Coroot working with it, the tables are not created automatically.

**Comment by apetruhin** [comment, marked as answer]
Coroot utilizes the [opentelemetry-collector](https://github.com/open-telemetry/opentelemetry-collector) to collect and save traces and logs to ClickHouse. You should also run it. It will create the necessary tables.

**Reply by T100D** [reply]
Looked at it, but i guess it requires a reinstall using helm and setting up cubernetes. 

Thank you.
